sdk/cosmosdb/Azure.ResourceManager.CosmosDB/api/Azure.ResourceManager.CosmosDB.netstandard2.0.cs

```diff
@@ -1694,6 +1747,40 @@ public partial class CassandraTableResourceInfo
         public Azure.ResourceManager.CosmosDB.Models.CassandraSchema Schema { get { throw null; } set { } }
         public string TableName { get { throw null; } set { } }
     }
+    public partial class ClientEncryptionIncludedPath
```

Comment:
Shall we add a `CosmosDB` prefix to this class name to specify?

Reply:
As long as the same naming format is used all across i am fine with it

Reply:
Is now CosmosDBClientEncryptionIncludedPath

Reply:
Does using rename-model: suffice. Do have a look at the changes made.

Reply:
#resolved